### ISSUE TITLE
docs rrd ruleset

### ISSUE THREAD

**Issue by rrd108** [rrd108/vue-mess-detector#142]
The docs say

"Opinionated rules that enforce a specific style of coding. These rules are not recommended for general use, but can be useful in specific cases."

I agree that this is true for **some** of the rules, but all of them?

@David-Pena ?

**Comment by David-Pena** [comment]
You are right, that description doesn't describe correctly what `rrd` rules are.

What do you think about this:

"Opinionated rules designed to enforce a consistent coding style. While some rules may be more strict and suited for specific scenarios, most of them can be broadly applied to enhance code quality across various projects."

**Comment by rrd108** [comment]
Sounds better, but I do not think that **all** of them are opiniated. Or not more opiniated then the official vue rules.

**Some** of them are very opiniated :smile: 

But I trust in your opinion as I see "my own" rules from a different perspective. But quite a few them are commonly accepted best practices and not the children of my mind.

**Comment by David-Pena** [comment]
> But I trust in your opinion as I see "my own" rules from a different perspective. But quite a few them are commonly accepted best practices and not the children of my mind.

No worries, let me try again 📝

I'll come back with a few options (I used opinionated because I've seen you describe your rules as that in some issues but you are right, there are just a few "controversial" ones)

**Comment by David-Pena** [comment]
What about this one:

Option A:
"A ruleset designed to enforce consistency and quality in your code, blending widely accepted standards with some more opinionated guidelines. It provides flexibility, allowing developers to choose the rules that best align with their project's style and goals."

Option B:
"A collection of rules that balances widely accepted best practices with more specific, opinionated guidelines, offering a comprehensive approach to code quality."

Or a mix: Option C
"A collection of rules that balances widely accepted best practices with more specific, opinionated guidelines, offering a comprehensive approach to code quality, allowing developers to choose the rules that best align with their project's style and goals."

**Comment by rrd108** [comment]
BTW should not we rename this ruleset to `vue-mess-detector`?

You also added some rule ideas, right? So it is no longer just mine ideas.

I think we should later add a `community` ruleset to accomodate other contributors new rules.

And I contacted some of the big guns if they want to add their rules.

**Comment by rrd108** [comment]
> Or a mix: Option C
> "A collection of rules that balances widely accepted best practices with more specific, opinionated guidelines, offering a comprehensive approach to code quality, allowing developers to choose the rules that best align with their project's style and goals."

🤩

**Comment by David-Pena** [comment]
> BTW should not we rename this ruleset to vue-mess-detector?

I don't think so, its a set which each rule was thought by you.

> You also added some rule ideas, right? So it is no longer just mine ideas.

I think I just added this one #141 but its technically from your description about another rule.

> I think we should later add a community ruleset to accomodate other contributors new rules.

I like this idea... I'll start thinking about new rules 📝

**Comment by David-Pena** [comment]
> 🤩

Is that a "send a commit with that description"?

**Comment by rrd108** [comment]
> > 🤩
> 
> Is that a "send a commit with that description"?

It is "thanks it sounds cool".

Commit if you want, if you have better to do I would do it.

**Comment by David-Pena** [comment]
Pushed to main @rrd108
